$OpenBSD: patch-akonadi_contact_CMakeLists_txt,v 1.1 2019/06/16 07:05:09 rsadowski Exp $

Fix build with Ninja 1.9.0.
Create only one custom target to generate contactactionssettings.h to avoid
duplicate ninja rules.

Index: akonadi/contact/CMakeLists.txt
--- akonadi/contact/CMakeLists.txt.orig
+++ akonadi/contact/CMakeLists.txt
@@ -38,6 +38,10 @@ set(akonadicontact_actions_SRCS
 
 configure_file( actions/contactactionssettings.kcfg.cmake ${CMAKE_CURRENT_BINARY_DIR}/contactactionssettings.kcfg @ONLY)
 kde4_add_kcfg_files(akonadicontact_actions_SRCS actions/contactactionssettings.kcfgc)
+add_custom_target(
+    generate_contactactionssettings_kcfg
+    DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/contactactionssettings.h
+)
 
 set(akonadicontact_editor_SRCS
   editor/addresseditwidget.cpp
